src/printapp/test/test-api.py

```diff
@@ -101,18 +101,14 @@ def test_upload_file(self):
             response = app.post('/api/upload', data=form_data)
             self.assertEqual(self._status(response), 400)
 
-    @unittest.skip("Skip test which actually prints.")
-    def test_print(self):
-        """Tests the /api/upload and /api/print endpoints.
-
-        This test is disabled by default. Enable it by commenting out the
-        decorator above.
+    def test_print_and_deletejob(self):
+        """Tests the /api/upload and /api/print and /api/deletejob/<job_id> endpoints.
         """
         with self.get_client() as app:
             self._sign_in(app)
 
             oid = []
-            file_names = ['test.pdf', 'test.txt', 'test.doc', 'test.docx']
```

Comment:
In any event, use fiddler to see why the tests are failing.

On Wed, May 6, 2015 at 7:48 PM, Tyler VanZanten <notifications@github.com>
wrote:

> In src/printapp/test/test-api.py
> <https://github.com/jglamine/calvinwebprint/pull/45#discussion_r29814078>:
>
> >          """
> >          with self.get_client() as app:
> >              self._sign_in(app)
> >
> >              oid = []
> > -            file_names = ['test.pdf', 'test.txt', 'test.doc', 'test.docx']
>
> Hmm... https://developers.google.com/cloud-print/docs/appInterfaces was
> last updated April 24, 2014.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jglamine/calvinwebprint/pull/45/files#r29814078>.
>